Doc/library/sys.rst

```diff
@@ -1276,8 +1276,10 @@ always available.
      :envvar:`PYTHONIOENCODING` environment variable before starting Python.
 
    * When interactive, standard streams are line-buffered.  Otherwise, they
```

Comment:
I think all this paragraph is related to stdout and stderr. Just replace "standard streams" with "stdout and stdin". No other changes are needed.

Reply:
> Just replace "standard streams" with "stdout and stdin".

Did you mean "stdout and stderr" instead of "stdout and stdin"?

Reply:
Yes. Sorry.

Reply:
Sure, I can change that. Another question is that shouldn't we still document stdin buffering?